recreate-fhir-in-dir.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# This bash script is used for applying the fhirdata primeCLI command to an entire directory.
+# This is useful when hl7 mappings need to be updated and this would cause a large number of fhir test output files to
+# need to be updated as well.
+# See ProcessFhirCommands.kt for more information on usage of the fhirdata primeCLI command.
+#
+# Example usage:
+#   (To be run from within the project root directory)
+#   bash ./recreate-fhir-in-dir.sh prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/
+#
+# Limitations: This script will update all differences including id fields, datetime fields, etc.
+# This can create a slightly noisy git diff. There may also be discrepancies with the json formatting between the
+# existing file and the output of this script. In that case use your best judgement and separate out formatting
+# changes into their own commits.
```

Comment:
If this is what you used, in theory the id references/urls should have been updated, but that didn't seem to happen?

Reply:
Yeah I left them out of my commits, and that's how it happened that I accidentally updated the resource.id and not the fullUrl on a number of them. I followed this approach because I think while it makes it a little easier on the implementer, it makes the review and file history that much messier and there is no benefit.